tests/testthat/test-ols-var.R

```diff
@@ -0,0 +1,25 @@
+set.seed(1)
```

Comment:
I would recommend changing the seed from something other than `1` to say `1246426` for example